### PR TITLE
chore: dashmate updates

### PR DESCRIPTION
**Pull request by strophy**
Testing has shown some edge cases in the system environment that can interfere with dashmate running. This PR documents steps users can take to avoid such issues.

Preview build: https://dash-docs--218.org.readthedocs.build/en/218/docs/user/masternodes/setup-testnet.html

### CODE REVIEW

**Comment by pshenmic** [comment]
What if you install AppArmor on regular images where it were working correctly without it before? Could it cause any issues?

DAPI single port is only applied to v24, that is not in production (master) yet. People manages v23 networks when install dashmate by default, and its still http-grpc ports for it.

File: docs/user/masternodes/setup-testnet.rst [hunk 1]
```diff
@@ -1238,8 +1244,7 @@ Your masternode is now providing service on the following local ports::
   Core RPC:     19998
   Platform P2P: 26656
   Platform RPC: 26657
-  DAPI HTTP:    3000
-  DAPI gRPC:    3010
+  DAPI:    3000
```
Comment (pshenmic) [inline docs/user/masternodes/setup-testnet.rst:1247]: This is changed only on v24, while this doc representing the testnet the suppose to be running v23 network.

**Review by pshenmic — CHANGES_REQUESTED** [review]

**Comment by thephez** [comment]
Thanks! Formatting and stuff all looks good. I'll  just wait for resolution on the open comments.

**Comment by strophy** [comment]
Pushed a change to clarify AppArmor is really optional and this step is for troubleshooting. Let's merge this once v0.24 hits testnet.

**Review by thephez — APPROVED** [review]
LGTM! :+1:

**Comment by strophy** [comment]
Yes, let's merge

**Review by thephez — APPROVED** [review]

**Pull request merged**